tools/aquainfra_importer/aquainfra_importer.xml

```diff
@@ -0,0 +1,13 @@
+<tool id="aquainfra_importer" name="AquaINFRA Importer" tool_type="data_source" version="1.0" profile="20.09">
+    <description>downloads content via the AquaINFRA interaction platform</description>
+    <command><![CDATA[
+        python '$__tool_directory__/data_source.py' '$output' $__app__.config.output_size_limit '$output'
```

Comment:
```suggestion
        python '$__tool_directory__/data_source.py' '$output' $__app__.config.output_size_limit
```

Additional params happen to be ignored by the `data_source.py` script so this was never used.